commons/com.b2international.index/src/com/b2international/index/revision/Conflict.java

```diff
@@ -15,13 +15,17 @@
  */
 package com.b2international.index.revision;
 
+import java.util.Objects;
+
 /**
  * @since 7.0
  */
 public abstract class Conflict {
 
 	private final ObjectId objectId;
 	private final String message;
+	
+	private ObjectId containerId;
```

Comment:
The new field is not included in `hashCode` - `equals` computations, is that allowed?